source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -232,7 +243,7 @@ This config was generated using SpongeForge build 2022 (with Forge 2202), Sponge
     # # If you need help with the configuration or have any questions related to Sponge,
     # # join us at the IRC or drop by our forums and leave a post.
     #
-    # # IRC: #sponge @ irc.esper.net ( https://webchat.esper.net/?channel=sponge )
+    # # IRC: #sponge @ irc.esper.net ( http://webchat.esper.net/?channel=sponge )
```

Comment:
Why don't we just have it link to the IRC/chat page on SpongeHome to keep them in the Sponge website ecosystem.